design/one-pager-component-mutable-and-versioning.md

```diff
@@ -465,17 +499,15 @@ spec:
     apiVersion: core.oam.dev/v1alpha2
     kind: ContainerizedWorkload
     name: frontend-a75588698
-status:
-  currentWorkloadRef:
-    apiVersion: core.oam.dev/v1alpha2
-    kind: ContainerizedWorkload
-    name: frontend-c8bb659c5
 ```
 
 Then the rollout trait will take control of the blue-green deploy progress.
 
-Finally the old workload will be removed by rollout trait and the status of rollout trait will align with the spec.
-Getting into another stable state.
+1. It will find out all controllerRevisions belongs to the component by label `controller.oam.dev/component=frontend`.
+2. With the controllerRevisions, it will find out all existing workload instance.
+2. It will start to do rolling update, decrease the replica of old workloads and increase the new workload. 
+
+Finally, the old workload will be removed by rollout trait, getting into another stable state.
```

Comment:
```suggestion
1. It will find out all the controllerRevisions that belong to the component with the label `controller.oam.dev/component=frontend`.
2. With the controllerRevision, it will find out all existing workload instances.
2. It will start to do a rolling update, decrease the replica of old workloads, and increase the new workload. 

Finally, the old workload will be removed by the rollout trait, getting into another stable state.
```